package.json

```diff
@@ -58,6 +58,7 @@
     "body-parser": "^1.14.1",
     "bucket-assets": "^1.0.2",
     "cheerio": "^0.19.0",
+    "coffee-loader": "^0.9.0",
```

Comment:
`coffee-loader` is likely pulling in `coffee-script` as a dep, and that's why it works -- but strictly speaking `coffee-loader` is only for webpack (and thus a client-side compile step, during development)